lib/matplotlib/backends/backend_ps.py

```diff
@@ -1440,27 +1440,32 @@ def convert_psfrags(tmpfile, psfrags, font_preamble, custom_preamble,
     command = '%s cd "%s" && latex -interaction=nonstopmode "%s" > "%s"'\
                 %(precmd, tmpdir, latexfile, outfile)
     verbose.report(command, 'debug')
-    exit_status = os.system(command)
-
-    with io.open(outfile, 'rb') as fh:
-        if exit_status:
+    try:
+        output = subprocess.check_output(command, shell=True,
+                                         stderr=subprocess.STDOUT)
+    except subprocess.CalledProcessError as exc:
+        with io.open(outfile, 'rb') as fh:
             raise RuntimeError('LaTeX was not able to process your file:\
-    \nHere is the full report generated by LaTeX: \n\n%s'% fh.read())
```

Comment:
This is (and was before) pretty ugly; please take advantage of automatic string concatenation for this:
```python
     raise RuntimeError('A string '
                        'continued on another line')
```